Content.Shared/StepTrigger/Systems/StepTriggerSystem.cs

```diff
@@ -77,8 +77,7 @@ private bool UpdateColliding(StepTriggerComponent component, TransformComponent
 
         if (otherPhysics.LinearVelocity.Length < component.RequiredTriggerSpeed
             || component.CurrentlySteppedOn.Contains(otherUid)
-            || otherAabb.IntersectPercentage(ourAabb) < component.IntersectRatio
-            || !CanTrigger(component.Owner, otherUid, component))
```

Comment:
This check already happens here: https://github.com/space-wizards/space-station-14/blob/a23abda84dac5390732887afc69b1c82dbea0e46/Content.Shared/StepTrigger/Systems/StepTriggerSystem.cs#L107

Reply:
Does HandleCollide trigger every tick?

From the code it looks like it fires HandleCollide then every tick it fires UpdateColliding until the entities stop colliding. CanTrigger also fires the StepTriggerAttemptEvent, which could change while the entity is still colliding, so this line is needed.